Sources/HederaProtobufs/Services/response_code.pb.swift

```diff
@@ -1796,7 +1796,7 @@ public enum Proto_ResponseCodeEnum: SwiftProtobuf.Enum {
 
 extension Proto_ResponseCodeEnum: CaseIterable {
   // The compiler won't synthesize support with the UNRECOGNIZED case.
-  public static var allCases: [Proto_ResponseCodeEnum] = [
+  public static let allCases: [Proto_ResponseCodeEnum] = [
```

Comment:
This change should be mirrored to `Sources/Hedera/Status.swift` (which is literally the same enum but for privacy reasons isn't actually that enum)